src/features/emails/components/EmailEditor/tools/InlineLink/index.tsx

```diff
@@ -0,0 +1,263 @@
+          this._focused = false;
+          this._api.inlineToolbar.close();
+          this.clear();
+        }
+      };
+
+      this._inputStatusMessage = document.createElement('div');
+      this._inputStatusMessage.textContent = this._config.messages.addUrl;
+      this._inputStatusMessage.style.display = 'none';
+      this._inputStatusMessage.style.fontSize =
+        this._config.theme.body2FontSize;
+      this._inputStatusMessage.style.paddingTop = '8px';
+
+      const svg = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
+      svg.setAttributeNS(null, 'viewBox', '0 0 24 24');
+      svg.style.display = 'block';
+      svg.style.height = '1.25rem';
+      svg.style.width = 'auto';
+      svg.style.marginRight = '8px';
+
+      const path = document.createElementNS(
+        'http://www.w3.org/2000/svg',
+        'path'
+      );
+      path.setAttributeNS(
+        null,
+        'd',
+        'M19 19H5V5h7V3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2v-7h-2v7zM14 3v2h3.59l-9.83 9.83 1.41 1.41L19 6.41V10h2V3h-7z'
+      );
+      path.setAttributeNS(null, 'stroke-width', '0');
+      path.setAttributeNS(null, 'fill', this._config.theme.mediumGray);
+      svg.appendChild(path);
+
+      const linkText = document.createElement('span');
+      linkText.textContent = this._config.messages.testLink;
+      linkText.style.fontSize = this._config.theme.body2FontSize;
+
+      const visitLinkContainer = document.createElement('div');
+      visitLinkContainer.style.display = 'flex';
+      visitLinkContainer.style.flexDirection = 'row';
+      visitLinkContainer.style.paddingTop = '8px';
+      visitLinkContainer.style.alignItems = 'center';
+      visitLinkContainer.appendChild(svg);
+      visitLinkContainer.appendChild(linkText);
+
+      this._visitLink = document.createElement('a');
+      this._visitLink.target = '_blank';
+      this._visitLink.style.textDecoration = 'none';
+      this._visitLink.style.color = 'inherit';
+      this._visitLink.appendChild(visitLinkContainer);
+
+      this._inputStatusContainer = document.createElement('div');
+      this._inputStatusContainer.appendChild(this._visitLink);
+      this._inputStatusContainer.appendChild(this._inputStatusMessage);
+
+      this._container = document.createElement('div');
+      this._container.style.padding = '8px';
+
+      this._container.appendChild(this._input);
+      this._container.appendChild(this._inputStatusContainer);
+
+      return this._container;
+    }
+
+    renderButton() {
+      this._button = document.createElement('button');
+      this._button.type = 'button';
+      this._button.classList.add(this._api.styles.inlineToolButton);
+      return this._button;
+    }
+
+    static get sanitize() {
+      return {
+        a: {
+          class: true,
+          href: true,
+          target: '_blank',
+        },
+      };
+    }
+
+    get shortcut() {
+      return 'CMD+Q';
```

Comment:
Isn't ⌘Q the keyboard shortcut to close apps in macOS?

Reply:
Yes it is! I thought we had changed this to CMD+K, thank you so much for identifying it. Will you fix this @ziggabyte?

Reply:
ooops i had it in my notes and overlooked it!! will fix and change to K!